server/views/pages/disclaimer.njk

```diff
@@ -26,29 +26,22 @@
           {{ govukCheckboxes({
```

Comment:
More work may be needed to prettify the UI / marry up with original - but need all working first